package.json

```diff
@@ -100,7 +100,8 @@
 		"ts-jest": "^29.0.5",
 		"ts-loader": "^9.4.2",
 		"typescript": "^4.9.4",
-		"vue-template-compiler": "~2.6"
+		"vue-template-compiler": "~2.6",
+		"webpack": "^5.75.0"
```

Comment:
unrelated change? webpack should come as peer of ``@nextcloud/webpack-vue-config``

Reply:
solved !